app.py

```diff
@@ -112,5 +112,6 @@ def logout():
 
 if __name__ == "__main__":
     app.run(
-        host=os.getenv("HOST", "0.0.0.0"), port=int(os.getenv("PORT", 8080)), debug=True
+        #host=os.getenv("HOST", "0.0.0.0"), port=int(os.getenv("PORT", 8080)), 
```

Comment:
I usually comment out those lines because when I run the app locally it takes forever to load versus me just doing debug=True. It's weird but it works for me on my end I usually try and uncomment what I commented before I push. I have also tried forwarding ports but it's not ass successful as the tedious uncomment and comment I do.

Reply:
Gotcha. Have you tried modifying the `HOST` environment variable in your `.env`?